### ISSUE TITLE
正则表达式工具可以增加一个

### ISSUE THREAD

**Issue by guilinonline** [CorentinTh/it-tools#1115]
### What type of request is this?

New tool idea

### Clear and concise description of the feature you are proposing

正则表达式工具
![image](https://github.com/CorentinTh/it-tools/assets/45056991/893ccec4-8833-4fa8-a9eb-2e3ea57d90ae)

### Validations

- [X] Check the feature is not already implemented in the project.
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Check that the feature can be implemented in a client side only app (IT-Tools is client side only, no server).

**Comment by sharevb** [comment]
Hi @guilinonline , could you explain in plain english ? thanks

**Comment by guilinonline** [comment]
Hi sharevb,
Let me walk you through this regular expression online testing interface:
At the top, there’s a large text box with the prompt “在此输入待匹配文本” — this is where you input the text you want to test against a regular expression.
Below that, there’s a section labeled “正则表达式”:
Enter your regular expression in the input box.
There are two checkboxes: “全局搜索” (enabled by default, meaning it will find all matches) and “忽略大小写” (if checked, case differences won’t affect matching).
Click the green “测试匹配” button, and the matching results will appear in the “匹配结果...” box.
Further down, there’s a “替换文本” section:
Enter the text you want to use for replacement in the input box.
Click the orange “替换” button, and the final replacement result will show up in the “替换结果...” box.
At the bottom, there’s a section for “常用正则表达式” with examples, like those for validating numbers, to help you get started.
If you need help testing a specific regex or understanding any part of the interface, just let me know!

**Comment by sharevb** [comment]
Hi @guilinonline , ok, so what you are looking for is: https://sharevb-it-tools.vercel.app/regex-tester

And if you are interested in an up to date version of it-tools, with many improvements, new tools, and bug fixes, as this repo is almost no more maintained, I made a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
